### PR TITLE
Editable Session Rates

### PR DESCRIPTION
**Pull request by wisemandenny**
This PR adds the ability to edit all fields in the sessions list view.
#376

### CODE REVIEW

File: frontend/src/components/sessions.js [hunk 1]
```diff
@@ -45,3 +91,10 @@ SessionsList.propTypes = {
         PropTypes.shape({ Header: PropTypes.any.isRequired })
     ),
 };
+
+export const ConnectedSessionsList = connect(
+    (state) => ({
+        sessions: sessionsSelector(state),
```
Comment (siefkenj) [inline frontend/src/components/sessions.js]: Let's switch this to the `useReducer` and `useDispatch` style.

Comment (wisemandenny) [reply frontend/src/components/sessions.js]: Do you mean `useSelector` from redux hooks or `useReducer` from React hooks?

Comment (siefkenj) [reply frontend/src/components/sessions.js]: From `Redux`

**Comment by wisemandenny** [comment]
~@siefkenj On this branch try editing and saving either rate on a session with the console open, you will see a "can't perform a state update on an unmounted component" error. I am not sure how to fix this, and also the same error doesn't appear when I edit the name field. Could you take a look at it and let me know if you know how to fix it? I am new to redux hooks.~

Just saw that you used `useSelector` in `editable-instructors-list.tsx`. I figured this issue out.

**Pull request closed without merging**